public/app/features/dashboard-scene/sharing/ShareExportTab.tsx

```diff
@@ -83,7 +83,7 @@ export class ShareExportTab extends SceneObjectBase<ShareExportTabState> {
       title = dashboardJson.title;
     }
     saveAs(blob, `${title}-${time}.json`);
-    trackDashboardSharingActionPerType('save_export', shareDashboardType.export);
+    reportInteraction('dashboards_sharing_export_download_json_clicked', { externally: isSharingExternally });
```

Comment:
It may be useful to add a trait so we know this is coming from the Scene feature?

Reply:
Maybe, but in all cases Scenes is using the same tracking as regular Dashboards so I'm doing the same.